openbb_platform/providers/intrinio/openbb_intrinio/models/etf_info.py

```diff
@@ -0,0 +1,662 @@
+        + " This may differ from where the ETF is domiciled or traded, particularly in Europe.",
+    )
+    issuer_country: Optional[str] = Field(
+        default=None,
+        description="2 letter ISO country code for the country where the issuer is located.",
+    )
+    domicile: Optional[str] = Field(
+        default=None,
+        description="2 letter ISO country code for the country where the ETP is domiciled.",
+    )
+    listing_country: Optional[str] = Field(
+        default=None,
+        description="2 letter ISO country code for the country of the primary listing.",
+        alias="listing_country_code",
+    )
+    listing_region: Optional[str] = Field(
+        default=None,
+        description="Geographic region in the country of the primary listing falls.",
+    )
+    bond_currency_denomination: Optional[str] = Field(
+        default=None,
+        description="For all bond ETFs, this field provides additional"
+        + " detail on the currency denomination of the underlying securities.",
+    )
+    base_currency: Optional[str] = Field(
+        default=None,
+        description="Base currency in which NAV is reported.",
+    )
+    listing_currency: Optional[str] = Field(
+        default=None,
+        description="Listing currency of the Exchange Traded Product (ETP) in which it is traded."
+        + " Reported using the 3-digit ISO currency code.",
+    )
+    number_of_holdings: Optional[int] = Field(
+        default=None,
+        description="The number of holdings in the ETF.",
+    )
+    month_end_assets: Optional[float] = Field(
+        default=None,
+        description="Net assets in millions of dollars as of the most recent month end.",
+    )
+    net_expense_ratio: Optional[float] = Field(
+        default=None,
+        description="Gross expense net of Fee Waivers, as a percentage of net assets"
+        + " as published by the ETF issuer.",
+        json_schema_extra={"x-unit_measurement": "percent", "x-frontend_multiply": 100},
+    )
+    etf_portfolio_turnover: Optional[float] = Field(
+        default=None,
+        description="The percentage of positions turned over in the last 12 months.",
+        json_schema_extra={"x-unit_measurement": "percent", "x-frontend_multiply": 100},
+    )
+
+
+class IntrinioEtfInfoFetcher(
+    Fetcher[IntrinioEtfInfoQueryParams, List[IntrinioEtfInfoData]]
+):
+    """Intrinio ETF Info Fetcher."""
+
+    @staticmethod
+    def transform_query(params: Dict[str, Any]) -> IntrinioEtfInfoQueryParams:
+        """Transform query."""
+        return IntrinioEtfInfoQueryParams(**params)
+
+    @staticmethod
+    async def aextract_data(
+        query: IntrinioEtfInfoQueryParams,
+        credentials: Optional[Dict[str, str]],
+        **kwargs: Any,
+    ) -> List[Dict]:
+        """Return the raw data from the Intrinio endpoint."""
+
+        api_key = credentials.get("intrinio_api_key") if credentials else ""
```

Comment:
api_key = credentials.get("intrinio_api_key", "")

Reply:
Every fetcher is like this currenctly: `api_key = credentials.get("intrinio_api_key") if credentials else ""`

Reply:
We can change it to this in another PR. :D